tools.js

```diff
@@ -613,7 +614,15 @@ module.exports = (() => {
 					banlistTable[toId(subformat.banlist[i])] = subformat.name || true;
 
 					let complexList;
-					if (subformat.banlist[i].includes('+')) {
+					if (subformat.banlist[i].includes('>')) {
+						complexList = subformat.banlist[i].split('>');
+						let limit = parseInt(complexList[1]);
+						let banlist = complexList[0].trim();
+						complexList = banlist.split('+').map(toId);
+						complexList.unshift(banlist, subformat.name, limit);
+						format.teamLimitTable.push(complexList);
+					}
+					else if (subformat.banlist[i].includes('+')) {
```

Comment:
In case the test error is unclear, this needs to be:

`} else if (subformat.banlist[i].includes('+')) {`

Otherwise, great change!